.eslintignore

```diff
@@ -5,3 +5,5 @@ workers/
 dist.js
 *.min.js
 mapbox-gl.js
+
+modules/core/src/utils/positions.ts
```

Comment:
What is this for?

Reply:
Sometimes eslint fix overwrites prettier fix in an incompatible way. Will try to remove the exception after this lands.